packages/browserify/test/config.js

```diff
@@ -58,34 +62,38 @@ test('Config override is applied if not specified and already exists at default
       module.exports = require('two')
     },
     configOverride: {
-    resources: {
-      three: {
-        packages: {
-          two: true
+      resources: {
+        three: {
+          packages: {
+            two: true
+          }
         }
       }
-    }
-  }
+    },
+    expectedResult: 30
   })
   const testResult = await runScenario({ scenario })
-  t.is(testResult, 30)
+  scenario.checkResult(t, testResult, scenario)
 })
 
 test('Config - Applies writeAutoPolicyDebug plugin option and dumps module object to disk', async (t) => {
   const scenario = createScenarioFromScaffold({
     opts: {
       writeAutoPolicy: true,
       writeAutoPolicyDebug: true
-    }
+    },
+    testType: 'falsy'
   })
   const { projectDir, policyDir } = await prepareScenarioOnDisk({ scenario, policyName: 'browserify' })
   const expectedPath = path.join(policyDir, 'policy-debug.json')
 
-  t.falsy(fs.existsSync(expectedPath), 'Module data does not yet exist')
+  scenario.checkResult(t, fs.existsSync(expectedPath), scenario)
```

Comment:
dont use checkResult to assert an unrelated value
just keep the original assertion